README.md

```diff
@@ -67,7 +67,7 @@ but the gist is as simple as using one of the following:
 
 ```
 # Bash
-$ rustup completions bash > /etc/bash_completion.d/rustup.bash-completion
+$ rustup completions bash > ~/.local/share/bash_completion/completions/rustup
```

Comment:
Does bash pick this up by default?  if so, from what version?

Reply:
`bash-completions` (the package that adds completions to `bash` and is a de-facto standard to the point that people don't realize that `bash` does not have this functionality built in) added this with this commit scop/bash-completion@c89dcbbd5510876f6304ef10806b00cc9fda19dc, which was included in release 2.2 on March 3rd, 2016 (https://github.com/scop/bash-completion/releases/tag/2.2).